ampd/src/cosmos.rs

```diff
@@ -164,20 +164,19 @@ mod tests {
                 info: "info".to_string(),
                 gas_wanted: 1000,
                 gas_used: 900,
-                tx: Some(prost_types::Any {
+                tx: Some(Any {
                     type_url: "axelar.type".to_string(),
                     value: vec![1, 2, 3, 4, 5, 6],
                 }),
                 timestamp: "timestamp".to_string(),
                 events: vec![abci::Event {
-                    kind: "event_type".to_string(),
+                    r#type: "event_type".to_string(),
```

Comment:
is this not going to break stuff?

Reply:
Nope, cuz I removed the `.into` in the end.